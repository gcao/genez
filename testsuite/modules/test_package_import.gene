#!/usr/bin/env gene run

# Test package imports (non-relative)
(import "math")        # Should find ./math.gene
(import "helpers")     # Should find ./helpers.gene
(import "basic")       # Should find ./math/basic.gene

(println "Testing package imports...")
(println "math/PI =" math/PI)
(println "math/square 5 =" (math/square 5))
(println "helpers/helper =" (helpers/helper "test"))
(println "basic/add 2 3 =" (basic/add 2 3))