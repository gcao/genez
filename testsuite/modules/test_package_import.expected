Testing package imports...
math/PI = 3.14159
math/square 5 = 25
helpers/helper = test
basic/add 2 3 = 5